docs/lib/examples/ModalNested.js

```diff
@@ -27,6 +28,12 @@ class ModalExample extends React.Component {
     });
   }
 
+  closeAllModals() {
+    this.setState({ nestedModal: !this.state.nestedModal }, () => {
```

Comment:
It would be best to just set these explicitly to `false`.
Also, the `setState` callback does not ensure the closing animation has completed or anything like that.